satpy/resample.py

```diff
@@ -1161,22 +1199,29 @@ class BucketSum(BucketResamplerBase):
     ----------
     fill_value : float (default: np.nan)
         Fill value for missing data
-    mask_all_nans : boolean (default: False)
-        Mask all locations with all-NaN values
+    skipna : boolean (default: True)
+        If True, skips NaN values for the sum calculation
+                    (similarly to Numpy's `nansum`). Buckets containing only NaN are set to zero.
```

Comment:
Maybe it is just github but the spacing on this docstring seems really weird. Everything should be aligned with the `If True` in the first sentence.

Reply:
Yep, I removed the indents and now the website builds.